packages/experiments-realm/blog-app.gts

```diff
@@ -399,6 +399,7 @@ class BlogAppTemplate extends Component<typeof BlogApp> {
         },
         meta: {
           adoptsFrom: ref,
+          realmURL: currentRealm.href,
```

Comment:
Is it a smell that we need to specify the realm 3 times between lines 402 and 407?

Reply:
Yeah it is, actually the realmURL in opts is there specifically for defining the target realm but the code in the card api didn't respect it in some cases. Fixed it in https://github.com/cardstack/boxel/pull/1942/commits/3d375c80927c9768adb832785367b19c093843c0 and removed the redundant `realmURL`